### DISCUSSION TITLE
How might one interact with an IPLD object/dag using iroh?

### DISCUSSION THREAD

**Discussion by robsyme** [n0-computer/iroh#651] [Q&A]
The [docs suggest](https://iroh.computer/docs/iroh-and-kubo) that iroh matches kubo for the feature "Interact with IPLD DAG nodes".

Is there any part of the documentation or codebase that might help somebody wanting to import and traverse IPLD objects and graphs using iroh?
Thanks!

**Comment by RangerMauve** [comment]
One thing I've been hoping we could talk about regarding this is the IPLD URL scheme and gateway endpoint that I've been working on. https://github.com/ipfs/specs/pull/293
